packages/@aws-cdk/aws-cloudfront/lib/index.ts

```diff
@@ -1,5 +1,7 @@
 export * from './cache-policy';
 export * from './distribution';
+export * from './public-key';
+export * from './key-group';
```

Comment:
(Nit) - Can you keep these alphabetized to satisfy my OCD? :)

Reply:
I can definitely fix this since I'm sorta OCD too :). For whatever reason I thought ordering mattered only in this file (I've seen it in new intex.ts files in the repo). I'll definitely alphabetize this.

Reply:
@njlynch While we're at OCD... should I address underscores in names for OAI and Web Distribution in a separate PR?

![image](https://user-images.githubusercontent.com/31543/106175931-ebbacd80-6153-11eb-81df-962ae2904675.png)

Reply:
Yes please. :D

Reply:
Done! https://github.com/aws/aws-cdk/pull/12752